site/shared/src/main/scala/shared/User.scala

```diff
@@ -3,7 +3,7 @@ package shared
 // TODO: is any of this actually used/needed by the client js code?
 
 case class User(
-  id:          Option[Long],
+  id:          Option[Int],
```

Comment:
My bad, I'm going to change the `id`'s type in the database so we can map it to a `Long` in our code and have more available ID's than using integers.